### PR TITLE
Replace `yoga-layout-prebuilt` with `yoga-wasm-web`

### PR DESCRIPTION
**Pull request by amitdahan**
# Summary

This is an attempt at tackling #150 by replacing [`yoga-layout-prebuilt`](https://github.com/vadimdemedes/yoga-layout-prebuilt) with [`yoga-wasm-web`](https://github.com/shuding/yoga-wasm-web).

(FWIW, relies on the recent #546 for top-level `await`)

### CODE REVIEW

File: src/styles.ts [hunk 1]
```diff
@@ -1,8 +1,34 @@
...
+	JUSTIFY_CENTER,
+	JUSTIFY_FLEX_END,
+	JUSTIFY_FLEX_START,
+	JUSTIFY_SPACE_AROUND,
+	JUSTIFY_SPACE_BETWEEN,
+	POSITION_TYPE_ABSOLUTE,
+	POSITION_TYPE_RELATIVE,
+	EDGE_START,
+	type YogaNode
+} from './yoga.js';
```
Comment (sindresorhus) [inline src/styles.ts:31]: This is a bit verbose. Maybe just `import * as Yoga ...`?

File: src/yoga.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+				createRequire(import.meta.url).resolve('yoga-wasm-web/package.json')
+			),
+			'dist/yoga.wasm'
+		)
+	)
+);
+
+export default Yoga;
+export * from 'yoga-wasm-web';
+export type {Node as YogaNode} from 'yoga-wasm-web';
```
Comment (sindresorhus) [inline src/yoga.ts:20]: I feel like `yoga-wasm-web` should handle all this boilerplate: https://github.com/shuding/yoga-wasm-web/issues/23

File: src/styles.ts (hunk 1 above)
Comment (amitdahan) [reply src/styles.ts]: Cool, applied

File: src/yoga.ts (hunk 2 above)
Comment (amitdahan) [reply src/yoga.ts:20]: Cool I see the issue is now closed thanks to @jeetiss in https://github.com/shuding/yoga-wasm-web/pull/24

Once that's released I'll apply it to this PR.

Comment (jeetiss) [reply src/yoga.ts:20]: it is released 🙌🏻 
[and works](https://github.com/vercel/satori/pull/429/commits/9a651dc1afdcf9b945d7af500298ab1503a16aeb)

**Comment by amitdahan** [comment]
Had to manually ignore `n/file-extension-in-import` for the `yoga-wasm-web/auto` imports.

Seems related to this issue: https://github.com/mysticatea/eslint-plugin-node/issues/258

File: src/yoga.ts (hunk 2 above)
Comment (amitdahan) [reply src/yoga.ts]: This is a lot cleaner now thanks to `/auto`, thanks @sindresorhus for raising this and @jeetiss for contributing it!

**Comment by vadimdemedes** [comment]
Thanks @amitdahan, great work!

**Pull request merged**